gpii/node_modules/eventLog/src/eventLog.js

```diff
@@ -228,73 +257,86 @@ gpii.eventLog.gotError = function (err, errType) {
 fluid.onUncaughtException.addListener(gpii.eventLog.gotError, "gpii-eventLog");
 
 /**
- * Gets the path of the new log file for this instance of gpii.
- * It can use one of the following environment variables to override the configured:
- *  GPII_EVENT_LOG: The path to the log file (a file, or tcp://<host>:<port>).
- *  GPII_EVENT_LOG_DIRECTORY: A directory where per-instance logs are kept
+ * Parses the log path (which can be overridden by the GPII_EVENT_LOG environment variable), and sets either the
+ * logServer or logPath member.
  *
  * @param {Component} that - The gpii.eventLog instance.
- * @return {String} The path to the new log file.
+ * @param {String} logPath - The path to the log file (a file, or a tcp socket in the format of `tcp://<host>:<port>`).
```

Comment:
See comment above about logPath vs logDestination

Reply:
See answer above about logPath vs logDestination.

Changed the variable name to reduce confusion.